pkg/interface/src/views/apps/notifications/graph.tsx

```diff
@@ -296,7 +295,7 @@ export function GraphNotification(props: {
     dm,
     singleAuthor
   );
-  const groupAssociation = useAssocForGroup(association.group);
+  const groupAssociation = useAssocForGroup(association?.group);
```

Comment:
I don’t have repro either. But do these functions handle undefined well?

Reply:
yes

Reply:
Well as per https://github.com/urbit/landscape/issues/799 it does appear to be this line in question. I just wonder when we are passing undefined as an association.

Reply:
I'm not sure, perhaps notifications aren't being cleaned up correctly on graph remove?